portal-web/test/functional/com/liferay/portalweb/macros/frontendinfrastructure/DataSetAdmin.macro

```diff
@@ -324,7 +324,7 @@ definition {
 			key_viewName = ${dataSetViewName});
 
 		if (isSet(key_fieldList)) {
-			DataSetAdmin.goToTab(tabName = "Fields");
+			DataSetAdmin.goToTab(tabName = "Visualization Modes");
```

Comment:
I don't think we have any list and cards poshi tests, so this should be enough, `Table` is the default inner tab.